build/tmp/compileJava/source-classes-mapping.txt

```diff
@@ -0,0 +1,8 @@
+com/wolox/training/models/Book.java
```

Comment:
Remember to erase this file (since you put the `build` directory in the `.gitignore` file) 😋